src/installer/managed/Microsoft.NET.HostModel/Microsoft.NET.HostModel.csproj

```diff
@@ -24,6 +25,17 @@
     <PackageReference Include="System.Text.Json" Version="$(SystemTextJsonVersion)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="$(CoreClrProjectRoot)tools\Common\Compiler\Win32Resources\ResourceData.cs" Link="Compiler\Win32Resources\ResourceData.cs" />
```

Comment:
Should I move those files? or Should I wait for his response?

Reply:
I think it would make sense to move these somewhere else - technically they would not even belong to "coreclr", but I don't think we have a better place right now. Also - it would make sense to change their namespace to something more generic - maybe just `Internal.Win32Resources`.

But I think it would be fine making that change in a separate PR - to not complicate this one too much with "maintenance" tasks.